tests/intern_functional_circle.js

```diff
@@ -8,44 +8,45 @@ define([
 ], function (intern, selectCircleTests) {
 
   intern.functionalSuites = selectCircleTests([
-    'tests/functional/sign_in',
-    'tests/functional/sign_in_cached',
-    'tests/functional/sign_in_blocked',
-    'tests/functional/sync_sign_in',
-    'tests/functional/sign_up',
-    'tests/functional/complete_sign_up',
-    'tests/functional/complete_sign_in',
-    'tests/functional/sync_sign_up',
-    'tests/functional/sync_v2_sign_up',
-    'tests/functional/sync_v2_sign_in',
-    'tests/functional/sync_v2_force_auth',
-    'tests/functional/mailcheck',
+    'tests/functional/avatar',
```

Comment:
Circle tests haven't been very flaky in the past few months. latest and stage OTOH 😬